sphinx_immaterial/apidoc/python/apigen.py

```diff
@@ -593,18 +596,18 @@ def object_description_transform(
     content = entity.content
     options = dict(entity.options)
     options["nonodeid"] = ""
-    all_entities_and_members = [
-        (entity, member),
-        *[
-            (
-                api_data.entities[sibling_member.canonical_object_name],
-                sibling_member if member is not None else None,
-            )
-            for sibling_member in (entity.siblings or [])
-        ],
-    ]
+    if member is not None:
+        all_members = [member] + member.siblings
+        all_entities = [api_data.entities[m.canonical_object_name] for m in all_members]
+    else:
+        all_entities = [
+            entity,
+            *(api_data.entities[s] for s in (entity.siblings or {})),
+        ]
+        all_members = [None for _ in all_entities]
```

Comment:
```suggestion
        all_members = [None] * len(all_entities)
```
No need to traverse the list if you just want a parallel list of `None` objects

Reply:
Done.